src/containerapp/azext_containerapp/_up_utils.py

```diff
@@ -339,7 +341,7 @@ def _create_service_principal(cmd, resource_group_name, env_resource_group_name)
         )
     sp = create_service_principal_for_rbac(cmd, scopes=scopes, role="contributor")
 
-    logger.info(f"Created service principal: {sp['displayName']}")
+    logger.warning(f"Created service principal: {sp['displayName']} with ID {sp['appId']}")
```

Comment:
Thanks for adding this is useful for investigations when GitHub workflow is failing with SP access permissions error. Please add the same when we are fetching & re-using an existing SP. Thanks!